std/process.d

```diff
@@ -3204,7 +3204,16 @@ private:
 
             // Cache the last call's result.
             static string lastResult;
-            if (v != lastResult) lastResult = v.idup;
+            if (v.empty)
+            {
+                // Return non-null array for blank result to distinguish from
+                // not-present result.
+                lastResult = "";
+            }
+            else if (v != lastResult)
+            {
+                lastResult = v.idup;
```

Comment:
No, he means swap the order of the if nesting. But I think the way you have it is fine